### PR TITLE
Odd shape

### PR DESCRIPTION
**Pull request by windwalker-888**

### CODE REVIEW

File: CrowsZA/functions/fn_scatterPatternOddPattern.sqf [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+		case 17: { _tmpI = _i + 20.8 + _offset * 7.4; 	_tmpJ = _j + 2 	+  _offset  	+	(10 * _row)};
+		case 18: { _tmpI = _i + 22.8 + _offset * 8.4; 	_tmpJ = _j + 13 +  _offset * 2 	+	(10 * _row)};
+		case 19: { _tmpI = _i + 24.8 + _offset * 9.4; 	_tmpJ = _j + 2 	+  _offset  	+	(10 * _row)};
+		case 20: { _tmpI = _i + 26.8 + _offset * 10.4; 	_tmpJ = _j + 0 	+   				(10 * _row);  _row = _row + 1 };
+		default {  };
+	};
+	
+	
+	
+	if (_tick == 20) then { _tick = -1};
```
Comment (Crowdedlight) [inline CrowsZA/functions/fn_scatterPatternOddPattern.sqf:62]: So if we got more than 20, it repeats the pattern? How do we ensure the units don't hit each other if they are getting put at exact same placement while repeating the pattern?

**Comment by Crowdedlight** [comment]
Please remove `` CrowsZA/addons/CrowsZA.pbo`` from the pull-request. And would be great if you could add .pbo to the .gitignore. Seems like I forgot about that, oops. :D

**Comment by windwalker-888** [comment]
> Please remove ` CrowsZA/addons/CrowsZA.pbo` from the pull-request. And would be great if you could add `.pbo` to the .gitignore. Seems like I forgot about that, oops. :D

oh ffs. yes i did. removing

File: CrowsZA/functions/fn_scatterPatternOddPattern.sqf (hunk 1 above)
Comment (Crowdedlight) [reply CrowsZA/functions/fn_scatterPatternOddPattern.sqf:62]: AHH missed that. Perfect.

**Pull request merged**